kuma/dashboards/templates/dashboards/includes/revision_dashboard_body.html

```diff
@@ -39,23 +38,46 @@
-                    <span class="revision_ip">
-                    <br/>IP: {{ revision_ip }}
-                    {% set ban_ip_url = '%s?%s' % (url('admin:core_ipban_add'), 'ip=%s' % revision_ip) %}
-                    <br/><a class="dashboard-ban-ip-link" href="{{ ban_ip_url }}" target="_blank">{{ _('Ban IP from Editing') }}</a>
-                    </span>
-                    {% endif %}
-                    {% if request.user.is_superuser and revision.creator.active_ban %}
-                        <div class="banned">
-                            banned {{ datetimeformat(revision.creator.active_ban.date, format='date') }}
-                            by {{ revision.creator.active_ban.by }}
-                        </div>
-                    {% endif %}
+                    <td class="dashboard-comment">
+                        {{ format_comment(revision) }}
                     </td>
+                    <td class="dashboard-author">
+                        <a href="{{ revision.creator.get_absolute_url() }}">{{ revision.creator }}</a>
+                        {% set revision_ip = revision.revisionip_set.first() %}
+                        {% if show_ips and revision_ip %}
+                        <span class="revision_ip">
+                            <br/>IP: {{ revision_ip.ip }}
+                            {% if revision_ip.user_agent %}<br/>User agent: {{ revision_ip.user_agent }}{% endif %}
+                            {% if revision_ip.referrer %}<br/>User agent: {{ revision_ip.referrer }}{% endif %}
+                            {% set ban_ip_url = '%s?%s' % (url('admin:core_ipban_add'), 'ip=%s' % revision_ip) %}
+                            <br/><a class="dashboard-ban-ip-link" href="{{ ban_ip_url }}" target="_blank">{{ _('Ban IP from Editing') }}</a>
+                        </span>
+                        {% endif %}
+                        {% if request.user.is_superuser and revision.creator.active_ban %}
+                            <div class="banned">
+                                banned {{ datetimeformat(revision.creator.active_ban.date, format='date') }}
+                                by {{ revision.creator.active_ban.by }}
+                            </div>
+                        {% endif %}
+                    </td>
+                    {% if show_spam_submission %}
+                    <td class="dashboard-spam">
+                        {% set submissions = revision.akismet_submissions.all() %}
+                        {% if submissions.count() > 0 %}
+                            <p>Sent as
+                            {% for submission in submissions %}
+                                {{ submission.type }}
+                                at {{ datetimeformat(submission.sent, format='datetime') }}
+                                by <a href="{{ submission.sender.get_absolute_url() }}">{{ submission.sender }}</a>{% if not loop.last %}, {% endif %}
```

Comment:
hm that's a bit nasty since it's an iteration over a list, I guess I could write it as a macro so at least `Sent as <list of submissions>` can be translated